en/messages.json

```diff
@@ -59,7 +59,7 @@
     "popupSignUpPanelContinue": {
         "message": "Continue"
     },
-    "popupRemainingAliases-2": {
+    "popupRemainingAliases_2": {
```

Comment:
also make sure  main repo matches this new message key

Reply:
Done: 
- https://github.com/mozilla/fx-private-relay-add-on/pull/249/files#r782532532
- https://github.com/mozilla/fx-private-relay-add-on/pull/249/files#diff-59a5db4e9a139708113dc0d0e60807e300b556f19ce611a5ffaa4a764c83f14b